src/chain_sync/tipset_syncer.rs

```diff
@@ -954,6 +965,9 @@ async fn sync_tipset<DB: Blockstore + Sync + Send + 'static>(
         return Err(e);
     }
 
+    // Call only once messages persisted
+    chain_store.put_delegated_message_hashes(proposed_head.block_headers().iter())?;
```

Comment:
```suggestion
chain_store.put_delegated_message_hashes(proposed_head.block_headers().iter())?;
```

Reply:
Do you think there's a superfluous comment here?

Reply:
Ah no, the diff lied to me. I thought there's an extra blank line. Ignore this.